src/vec-final.md

```diff
@@ -47,7 +47,7 @@ impl<T> RawVec<T> {
 
             // If allocate or reallocate fail, oom
```

Comment:
This PR missed comment modification, see https://github.com/rust-lang-nursery/nomicon/pull/77

Reply:
Do you want to submit a comment fix?

Reply:
I'm a bit time-constrained for now, there may be other places to check.